class.jetpack.php

```diff
@@ -4544,24 +4544,14 @@ public static function register() {
 		);
 		$response = Jetpack_Client::_wp_remote_request( Jetpack::fix_url_for_bad_hosts( Jetpack::api_url( 'register' ) ), $args, true );
 
-
 		// Make sure the response is valid and does not contain any Jetpack errors
-		$valid_response = Jetpack::init()->validate_remote_register_response( $response );
-		if( is_wp_error( $valid_response ) || !$valid_response ) {
-		    return $valid_response;
+		$json = Jetpack::init()->validate_remote_register_response( $response );
```

Comment:
Should we call this variable `$json`? It appears to be a PHP object since we've decoded it in the previous object.